src/Adapter/Product/Combination/QueryHandler/GetCombinationForEditingHandler.php

```diff
@@ -159,7 +160,8 @@ public function handle(GetCombinationForEditing $query): CombinationForEditing
             $this->getDetails($combination),
             $this->getPrices($combination, $product),
             $this->getStock($combination),
-            $this->getImages($combination)
+            $this->getImages($combination),
+            $defaultCombinationId && $defaultCombinationId->getValue() === (int) $combination->id
```

Comment:
Just curious but is it possible that `$defaultCombinationId` is empty? I thought it was always specified
The only case when it could be empty I can figure is when the product has no combinations 🤔. But then we can't edit the combination so this query shouldn't be used as well

Anyway additional safety doesn't hurt, but I was curious to know if you really encountered an issue with this one?

Reply:
I didn't encounter any issues, but the method getDefaultCombination can return null this safety check is a must (at least so phpstan g

Reply:
Ok great